.github/workflows/release.yml

```diff
@@ -40,12 +40,13 @@ jobs:
         if: ${{ success() }}
         run: echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
 
-      - name: Create zip for sdkman
+      - name: Create zip for dependency managers
         if: ${{ success() }}
         run: |
           cd ktlint-cli/build/run
+          # Doing this for Homebrew and https://github.com/sdkman/sdkman-cli/wiki/Well-formed-SDK-archives
           mkdir -p ktlint-${{ env.version }}/bin
-          cp ktlint ktlint-${{ env.version }}/bin
+          cp ktlint ktlint-${{ env.version }}/bin/ktlint-${{ env.version }}
```

Comment:
I just see that the name of the binary is also suffixed with the version number. That does not seem to be a requirement according to https://github.com/sdkman/sdkman-cli/wiki/Well-formed-SDK-archives. Also, I wonder what the effect could be for the `homebrew` dependency. Please explain why the version number is required in the binary.

Reply:
You're right. I was looking at a few other projects and they were doing that, but it isn't a requirement. I'll roll that back.